fido/client.py

```diff
@@ -0,0 +1,21 @@
+def _twisted_web_client():
+    from twisted.web import client
+    return client
+
+
+class HTTP11ClientProtocolOverride(_twisted_web_client().HTTP11ClientProtocol):
```

Comment:
@lucagiovagnoli This file was made "private" and a warning was attached to prevent people from mistakenly importing it. Furthermore, a `from fido import *` will not import the file.

We need to access the Twisted API to instrument the TCP_NODELAY, so the `web.client` import is unavoidable. If you have any suggestions, I'd be happy to make the changes.